assets/activation-scripts/activate

```diff
@@ -19,23 +24,14 @@
 export _flox_activate_tracelevel="${_FLOX_PKGDB_VERBOSITY:-0}"
 [ "$_flox_activate_tracelevel" -eq 0 ] || set -x
 
-# These all derive from the `flox-interpeter` package
-export _activate_d="@out@/activate.d"
-export _profile_d="@out@/etc/profile.d"
-export _tcsh_home="@out@/activate.d/tcsh_home"
-export _zdotdir="@out@/activate.d/zdotdir"
-
-# External tools
-export _bash="@bash@"
-export _coreutils="@coreutils@"
-export _daemonize="@daemonize@"
-export _flox_activations="@flox_activations@"
-export _getopt="@getopt@"
-export _gnused="@gnused@"
-export _jq="@jq@"
-export _nawk="@nawk@"
-export _process_compose="@process-compose@"
-export _setsid="@setsid@"
```

Comment:
glad to see these not being globals anymore

Reply:
Thanks! Always knew we had an env variable leakage epic in our future. 😄